pages/_document.js

```diff
@@ -23,40 +12,24 @@ export default class Document extends NextDocument {
         <Head />
         <body>
           <Main />
-          {process.env.NODE_ENV !== "production" ? (
+          <template id="scripts">
```

Comment:
Rendering the `<script>` elements within a `<template>` element makes it unnecessary to escape anything. That’s what the `<template>` element is for: storing DOM fragments.